### DISCUSSION TITLE
Gothic 2 Steam Fix (International)

### DISCUSSION THREAD

**Discussion by Kor-Angar** [dosinabox/gothic2_steam_fix#23] [General]
если ставить на чистую готику 2 НВ создастся парадокс :-)
![image](https://user-images.githubusercontent.com/49514764/152853355-8ce936f5-c910-401c-85b5-2f7c4ecc37c0.png)

надо наверное добавить файл ini

**Comment by dosinabox** [comment]
> надо наверное добавить файл ini

Он должен быть там по умолчанию, если не используется бета original release.

**Reply by Kor-Angar** [reply]
я такой ставил Gothic_2_Steam_Fix_12.2021

**Reply by dosinabox** [reply]
![image](https://user-images.githubusercontent.com/39790055/152854047-b2931152-8ddc-499a-86d2-076fb6aabff6.png)

**Reply by Kor-Angar** [reply]
а, ты про сам стим. я игру ставил из дистрибутива :-)

**Reply by dosinabox** [reply]
В таком случае много чего добавлять придется. Мой сборник - он для стима, где издатель уже добавил некоторые полезные вещи, которые мне больше не нужно включать. Например, 4гб патч.

**Reply by Kor-Angar** [reply]
оно как бы да. но на форуме многократно писали что стим фикс ставится и на версию с диска. 
***
В любом случае если так задумано то это не ошибка, а так задумано.

**Comment by Kor-Angar** [comment]
у тебя туда юнион включён?

**Reply by dosinabox** [reply]
да
